5439 clarify defaultAddonName option and mention to the generator readme the creation of the theme add-on (#5709)
plone/volto: packages/generator-volto/README.md

# Yeoman Volto App Generator

> @plone/generator-volto is a Yeoman generator that helps you to set up Volto via the command line.

Reviewer: ```suggestion
@plone/generator-volto is a Yeoman generator that helps you to set up a Volto project via the command line.
```

npm init yo @plone/volto
```

> This is the shortcut for using `npm init` command. It uses Yeoman (`yo`) and `@plone/generator-volto` and `npm` executes the commands without having the need for the packages to be installed globally.

Reviewer: npm or pnpm?

```suggestion
This is the shortcut for using `npm init` command.
It uses Yeoman (`yo`) and `@plone/generator-volto`, and `npm` executes the commands without having the need for the packages to be installed globally.
```

Author: @stevepiercy npm is correct, pnpm is used only for the development of volto core.
When you install the volto projects it's still npm only and yarn to run :) 
Personally, I would remove this entire section because before it we mentioned commands
to install both items globally so why would we confuse beginners to then mention oh
you can use npm init without the need of what we've asked you todo previously?

Reviewer: I agree. A README should be a linear process. Extra stuff should go into documentation, or at the very end of the README if there are no docs, as an addendum.

> Answer the prompt questions to complete the generation process.

Reviewer: ```suggestion
Answer the prompts to complete the generation process.
```

```

> This will bootstrap a new Volto project inside the current folder. It will ask

Reviewer: ```suggestion
This will bootstrap a new Volto project inside the current folder.
```

> a few questions: project name, project description and a list of add-ons. 

Reviewer: ```suggestion
It will ask a few questions for the project name, project description, and a list of add-ons. 
```

to see a full list of options and arguments.

> [!NOTE]  

> > Besides the Volto project the generator also generates a theme add-on inside `src/addons`

Reviewer: ```suggestion
> In addition to the Volto project, the generator creates a theme add-on inside `src/addons` with the default name of `volto-[<project-name>]`.
```

> > with the default name of `volto-[<project-name>]`.

Reviewer: ```suggestion
```

> You can provide a specific name for the theme add-on that is created on project generation like:

> > ```bash

Reviewer: Let's make it generic.

```suggestion
> ```shell
```

> yo @plone/volto --defaultAddonName="volto-project-theme"
> ```

> > If you've used the example project name `myvoltoproject` inside `src/addons` you will find the Volto theme add-on named `volto-project-theme` instead of the default generator name `volto-myvoltoproject`.

Reviewer: ```suggestion
> If you've used the example project name `myvoltoproject` inside `src/addons`, you will find the Volto theme add-on named `volto-project-theme` instead of the default generator name `volto-myvoltoproject`.
```
